tutorial2_quantities_and_amounts/README.md

```diff
@@ -0,0 +1,59 @@
+# Tutorial 2
+
+The goals for this tutorial are:
+
+- to get comfortable with the [Github workflow](../GIT_SETUP.md), and with using their [local development environment](../README.md).
+- to understand how to load in a dataset using [d3-fetch](https://github.com/d3/d3-fetch) (d3.json, d3.csv, etc.).
```

Comment:
I would remove these and add `d3-axis` as a goal

Reply:
wanted to express that we're still practicing the local host so I updated it a little